### PR TITLE
MODPUBSUB-193 Fix tests after RMB update

### PR DESCRIPTION
**Pull request by bisecomsergiy**
## Purpose
In the scope of the task on module tests fix after upgrade, proper actions were fulfilled

## Approach
- Corrected error statuses in tests
- Moved models from client and remained them in server module (turned off interfaces creation for client)
- Added commons-lang artifact
- Removed guava dependency

The mod operation is tested by:
Post request:
pubsub post 201
Get request:
pubsub 200

## Learning
https://issues.folio.org/browse/MODPUBSUB-193

### CODE REVIEW

File: mod-pubsub-client/pom.xml [hunk 1]
```diff
@@ -208,12 +196,23 @@
         <groupId>org.folio</groupId>
         <artifactId>domain-models-maven-plugin</artifactId>
         <version>${raml-module-builder.version}</version>
```
Comment (SerhiiNosko) [inline mod-pubsub-client/pom.xml:198]: Do we need this plugin here if it is not generating clients and interfaces?

File: mod-pubsub-client/src/test/java/org/folio/util/pubsub/PubSubClientUtilsTest.java [hunk 2]
```diff
@@ -13,6 +13,7 @@
 
 import org.folio.util.pubsub.support.DescriptorHolder;
 import org.junit.After;
+import org.junit.Ignore;
```
Comment (SerhiiNosko) [inline mod-pubsub-client/src/test/java/org/folio/util/pubsub/PubSubClientUtilsTest.java:16]: We dont need to import  org.junit.Ignore

File: mod-pubsub-client/pom.xml [hunk 3]
```diff
@@ -198,7 +183,10 @@
             <phase>initialize</phase>
             <configuration>
               <sources>
-                <source>${project.build.directory}/generated-sources</source>
+                <!-- Adds client classes generated in current client module-->
+                <source>${project.build.directory}/generated-sources/raml-jaxrs</source>
```
Comment (SerhiiNosko) [inline mod-pubsub-client/pom.xml:187]: clients not generating here, so we dont need to include them

**Review by SerhiiNosko — APPROVED** [review]

File: mod-pubsub-client/pom.xml (hunk 1 above)
Comment (bisecomsergiy) [reply mod-pubsub-client/pom.xml]: Moved the plugin, thank you

File: mod-pubsub-client/src/test/java/org/folio/util/pubsub/PubSubClientUtilsTest.java (hunk 2 above)
Comment (bisecomsergiy) [reply mod-pubsub-client/src/test/java/org/folio/util/pubsub/PubSubClientUtilsTest.java]: Thanks, it is moved

File: mod-pubsub-client/pom.xml (hunk 3 above)
Comment (bisecomsergiy) [reply mod-pubsub-client/pom.xml]: The line is moved as well, thanks

**Review by RuslanLavrov — APPROVED** [review]

File: NEWS.md [hunk 4]
```diff
@@ -2,6 +2,7 @@
 * [MODPUBSUB-186](https://issues.folio.org/browse/MODPUBSUB-186) Provide 'ssl.endpoint.identification.algorithm' property
 * [MODPUBSUB-187](https://issues.folio.org/browse/MODPUBSUB-187) Add support for max.request.size configuration for Kafka messages
 * [KAFKAWRAP-2](https://issues.folio.org/browse/KAFKAWRAP-2) Take folio-kafka-wrapper lib out of mod-pubsub repository
+* [MODPUBSUB-193](https://issues.folio.org/browse/MODPUBSUB-193) Fix tests after RMB update
```
Comment (KaterynaSenchenko) [inline NEWS.md:5]:
```suggestion
* [MODPUBSUB-189](https://issues.folio.org/browse/MODPUBSUB-189) Upgrade to RAML Module Builder 33.0.2
* [MODPUBSUB-193](https://issues.folio.org/browse/MODPUBSUB-193) Fix tests after RMB update
```

**Review by KaterynaSenchenko — APPROVED** [review]

File: NEWS.md (hunk 4 above)
Comment (bisecomsergiy) [reply NEWS.md:6]: The line is added, thank you

**Pull request merged**